src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -3985,7 +3985,7 @@ void DynamicIntConvSurfaceClassification(EnergyPlusData &state, int const SurfNu
     } else { // is controlled, lets see by how and if that means is currently active
 
         auto &zoneEquipConfig = state.dataZoneEquip->ZoneEquipConfig(surface.Zone);
-        auto &zoneNode = state.dataLoopNodes->Node(zone.SystemZoneNodeNumber);
+        auto const &zoneNode = state.dataLoopNodes->Node(zone.SystemZoneNodeNumber);
```

Comment:
[src/EnergyPlus/ConvectionCoefficients.cc:3052]:(style),[constVariable],Variable 'zone' can be declared as reference to const
[src/EnergyPlus/ConvectionCoefficients.cc:3988]:(style),[constVariable],Variable 'zoneNode' can be declared as reference to const